mmcls/models/backbones/shufflenet_v1.py

```diff
@@ -249,18 +251,20 @@ def _freeze_stages(self):
                 param.requires_grad = False
 
     def init_weights(self):
-        super(ShuffleNetV1, self).init_weights()
-        for name, m in self.named_modules():
-            if isinstance(m, nn.Conv2d):
-                if 'conv1' in name:
-                    normal_init(m, mean=0, std=0.01)
-                else:
-                    normal_init(m, mean=0, std=1.0 / m.weight.shape[1])
-            elif isinstance(m, (_BatchNorm, nn.GroupNorm)):
-                constant_init(m.weight, val=1, bias=0.0001)
-                if isinstance(m, _BatchNorm):
-                    if m.running_mean is not None:
-                        nn.init.constant_(m.running_mean, 0)
+        if self.init_cfg:
+            super(ShuffleNetV1, self).init_weights(self.init_cfg)
```

Comment:
If the users pass `init_cfg=None` to `ShuffleNet`, this method will continue to use the default init method. It will be confusing.